index.html

```diff
@@ -0,0 +1,92 @@
+          <a href="index.html" class="active">Home</a>
+          <a href="pages/facts.html">Fun Facts</a>
+          <a href="pages/gallery.html">Gallery</a>
+          <a href="pages/about_us.html">About Us</a>
+        </nav>
+    </header>
+    
+    <main>
+      <h1> &#9749; For the Love of Coffee &#9749; 
+      </h1>
+      <section>
+          <img id="logo" src="./images/logo.gif" alt="logo">
+        
+          <h2> A dedicated place for coffee appreciation and its cultural significance.</h2>
+      </section>
+      
+      <div class="grid-container">
+        <section id="description1">
+          <h3> What <em>is</em> coffee? </h3>
+            <p>
+              Ahhhh, coffee. Coffee is an unofficial sponsor of weekdays.
+              It can turn the groggiest person into an alert and active one!
+            </p>
+            <p>
+              Okay, in all seriousness, coffee in our everyday sense is a beverage brewed from roasted coffee beans, and is prepared in several different ways
+              all across the world. Some people prefer it plain, while others prefer to add milk, cream, sugar, spices, and several other additions.
+              However, the National Coffee Association defines coffee as the berries harvested from a tropical plant.
+            </p>
+            <p>
+              Visit our Fun Facts page to learn more neat details about cofee. You'll also
+              find a neat collection of our favorite coffeehouses in our Gallery page!
+              To learn more about the people behind this fansite, dive into our About Us page.
+            </p>
+            <img src="./images/coffee_beans.png" alt="Coffee beans clipart">
+        </section>
+        <section id="description2">
+          <h4>Origins</h4>
+            <p>
+              The when and how coffee was discovered is largely unknown. Several stories and legends 
+              exist to explain its origins. One thing that is for certain, is all coffee grown around the globe can be traced
+              back to modern-day Ethiopia.
+            </p>
+            <p>
+              There is a legend dated back to the third century AD, telling us how a goat shepherd noticed the change in behavior of his
+              goats after they consumed these "energizing" berries. Over time, during the 15th century, knowledge of the energizing "berries" spread from 
+              Ethiopia to the Arabian peninsula, Yemen to be exact, where coffee was first cultivated and domesticated. Here, the coffee trade 
+              and human caffeination took off.
+            </p>
+            <p>
+              Speaking of caffeination, the energizing properties of coffee are due to a special organic molecule named <em>caffeine</em>.
+              As a stimulant, caffeine acts on the Central Nervous System which affects a person's cognitive function.
+            </p>
+            <img src="./images/Ethiopia_map.png" alt="Ethiopia Map clipart">
```

Comment:
I would consider adding more descriptive `alt` messages for users that are using screen readers and when the images don't load.